runtime/fundamentals/node.md

```diff
@@ -16,12 +16,20 @@ oldUrl:
 - /runtime/manual/node/private_registries
 ---
 
-Modern Node.js projects will run in Deno with little to no reworking required.
-However, there are some key differences between the two runtimes that you can
-take advantage of to make your code simpler and smaller when migrating your
-Node.js projects to Deno.
+- **Deno is fully Node-compatible**. Virtually any modern Node project will run
+  in Deno with little or no change!
+- **Deno supports virtually all npm packages**. Just use the `npm:` specifier in
+  the import, and Deno takes care of the rest.
```

Comment:
I'd also remove the"virtually" caveat here from this headline. We can expand into talking about some limitations in the more detailed part of the page, but the headline here is "you can use npm packages in deno, and look at the simple syntax".

How about:

```suggestion
- **Deno supports npm packages**. Just use the `npm:` specifier in
  the import, and Deno takes care of the rest.
```